service/src/main/java/com/bankbazaar/service/manager/TripActivityService.java

```diff
@@ -22,40 +18,29 @@ public class TripActivityService {
     private UserService userService;
     @Autowired
     private TripActivityManager manager;
-    @Autowired
-    private TripUserMapManager tripUserMap;
-    public ModelAndView createActivityService(Long tripId){
-        ModelAndView model = new ModelAndView("create_activity");
-        model.addObject("tripId", tripId);
-        return model;
-    }
 
-    public String addTripService(Long tripId, TripActivityDto tripActivity, Principal principal) {
+
+    public TripActivityDto addTripService(Long tripId, TripActivityDto tripActivity, Principal principal) {
```

Comment:
Your service methods should not be appended with service.
Example addTripService should be addTrip